packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -47,6 +47,7 @@ export interface SelectProps<T = string | number> {
   width?: 'sm' | 'md' | 'lg' | 'full'
   inputHeight?: InputProps['height']
   onChange: (value: T) => void
+  onOpenChange?: (isOpen: boolean) => void
```

Comment:
lets rename this to `onOpen` since it indicates `on` indicates there is a handler associated